modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/CommitWriteInvokeClosure.java

```diff
@@ -80,6 +80,13 @@ public void call(@Nullable VersionChain oldRow) throws IgniteInternalCheckedExce
         RowVersion current = storage.readRowVersion(oldRow.headLink(), DONT_LOAD_VALUE);
         RowVersion next = oldRow.hasNextLink() ? storage.readRowVersion(oldRow.nextLink(), DONT_LOAD_VALUE) : null;
 
+        if (next == null && current.isTombstone()) {
+            // Previous version doesn't exist and current version is a tombstone, delete current version.
```

Comment:
I would paraphrase in:
`If only one version, and it is a tombstone, then we will remove such a chain.`